src/swap/types.ts

```diff
@@ -121,6 +131,7 @@ export type SwapButtonReact = {
 };
 
 export type SwapContextType = {
+  address?: Address;
```

Comment:
Probably we want a line of comment here explain when the `address` here is used.